packer/ubuntu-14.04_amd64-droplet.json

```diff
@@ -56,11 +53,11 @@
     }
   ],
   "push": {
-    "name": "capgemini/apollo-mesos-ubuntu-14-04-amd64"
+    "name": "Enxebre/apollo-mesos-ubuntu-14-04-amd64"
```

Comment:
i like this way of doing it for vagrant boxes http://docs.vagrantup.com/v2/boxes/versioning.html, so we could just manage it in the vagrant file. i know this is for elastic map reduce, but surely would be the same for ec2 http://docs.aws.amazon.com/ElasticMapReduce/latest/DeveloperGuide/emr-ami-version-numbers.html